examples/glfw/CMakeLists.txt

```diff
@@ -15,6 +15,15 @@ add_subdirectory(${CMAKE_SOURCE_DIR}/../../../third_party/glfw glfw)
 target_link_libraries(flutter_glfw glfw)
 include_directories(${CMAKE_SOURCE_DIR}/../../../third_party/glfw/include)
 
+
+############################################################
+# SKIA
+############################################################
+option(SKIA_INSTALL "" OFF)
+add_subdirectory(${CMAKE_SOURCE_DIR}/../../../third_party/skia/out/config skia)
+target_link_libraries(flutter_glfw skia)
+include_directories(${CMAKE_SOURCE_DIR}/../../../third_party/skia)
```

Comment:
To keep the build simple, I'd avoid building Skia here and just implement the rectangle behavior you need in the example.

Reply:
That sounds like a good approach! Thanks!